testcases/kernel/syscalls/fanotify/fanotify11.c

```diff
@@ -63,6 +63,9 @@ void test01(unsigned int i)
 
 	fd_notify = fanotify_init(tcases[i], 0);
```

Comment:
What about using `SAFE_FANOTIFY_INIT` instead?

Reply:
OK, could move the `FAN_REPORT_TID` check to `safe_fanotify_init()`, then fanotify11 could also use `SAFE_FANOTIFY_INIT`

I'll test it and update this patch.